osidb/models.py

```diff
@@ -2927,6 +2928,22 @@ def is_rhscl(self) -> bool:
         """
         return self.bts_key == RHSCL_BTS_KEY
 
+    @property
+    def y_stream(self):
```

Comment:
I am thinking whether it would be better to call the property `y_streams` when they can be more. I am not strongly for it but it would more naturally describe what can one await as a result.

Reply:
I think it makes sense, I will change to plural.